tests/main.test.ts

```diff
@@ -260,6 +260,11 @@ it(`should allow to call "corepack install" without arguments within a configure
 });
 
 it(`should refuse to run a different package manager within a configured project`, async () => {
+  // To ensure we test the default behavior, we must remove these env vars
+  // in case the local machine already set these values.
+  delete process.env.COREPACK_ENABLE_STRICT;
+  delete process.env.FORCE_COLOR;
```

Comment:
We should move it to [`setupTests.js`](https://github.com/nodejs/corepack/blob/main/tests/setupTests.js), to ensure a consistent environment for all tests.

Reply:
Good idea! I pushed an update 👍